src/main/java/com/google/devtools/build/lib/bazel/rules/android/AndroidNdkRepositoryFunction.java

```diff
@@ -209,6 +210,17 @@ private static String createCcToolchainRule(
         .replace("%toolchainFileGlobs%", toolchainFileGlobs.toString().trim());
   }
 
+  private static String getPlatformCpuLabel(String targetCpu) {
+    // Create a mapping of CcToolchain CPU values to platform arch constraint values
+    // in @bazel_tools//platforms
+    switch (targetCpu) {
+      case "x86": return "x86_32";
+      case "armeabi-v7a": return "arm";
+      case "arm64-v8a": return "aarch64";
+      default: return "x86_64";
```

Comment:
hmm, should we throw an error instead of defaulting to x86_64? e.g. if someone tries to select mips, or has a typo?